src/components/Widgets/ResponsiveUtils/responsiveUtils.js

```diff
@@ -16,14 +16,14 @@ class ResponsiveUtil {
     }
 
     viewPortSelector(){
-        if(window.innerWidth > breakpoints.desktop){
-            this.viewPort='Desktop'
```

Comment:
Try to return viewport in another fashion. Concern me first.

Reply:
Please suggest what needs to be done in that

Reply:
This is hardcoded, instead try to use a flag, or import the items from another file or use a closure.